aks-kubesphere-linux/nested/AKS.json

```diff
@@ -0,0 +1,188 @@
+            },
+            "minValue": 0,
+            "maxValue": 1023
+        },
+        "kubernetesVersion": {
+            "type": "string",
+            "defaultValue": "1.18.14",
+            "metadata": {
+                "description": "Kubernetes 的版本。"
+            }
+        },
+        "MasterNodeCount": {
+            "type": "int",
+            "metadata": {
+                "description": "The number of Master node."
+            }
+        },
+        "WorkerNodeCount": {
+            "type": "int",
+            "defaultValue": 3,
+            "metadata": {
+                "description": "The number of Worker node."
+            }
+        },
+        "networkPlugin": {
+            "type": "string",
+            "allowedValues": [
+                "azure",
+                "kubenet"
+            ],
+            "defaultValue": "azure",
+            "metadata": {
+                "description": "用于构建 Kubernetes 网络的网络插件。"
+            }
+        },
+        "vnetSubnetID": {
+            "type": "string"
+        },
+        "serviceCidr": {
+            "type": "string"
+        },
+        "dnsServiceIP": {
+            "type": "string"
+        },
+        "dockerBridgeCidr": {
+            "type": "string"
+        },
+        "enableRBAC": {
+            "type": "bool",
+            "defaultValue": true,
+            "metadata": {
+                "description": "用于打开和关闭 RBAC 的布尔型标志。"
+            }
+        },
+        "vmssNodePool": {
+            "type": "bool",
+            "defaultValue": true,
+            "metadata": {
+                "description": "用于打开和关闭虚拟机规模集的布尔型标志"
+            }
+        },
+        "windowsProfile": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "用于打开和关闭虚拟机规模集的布尔型标志"
+            }
+        },
+        "enablePrivateCluster": {
+            "type": "bool",
+            "defaultValue": true,
+            "metadata": {
+                "description": "启用对 Kubernetes 群集的专用网络访问。"
+            }
+        },
+        "enableAzurePolicy": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "用于启用和禁用 Azure Policy 加载项的布尔型标志。"
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_F8s_v2",
+            "metadata": {
+                "description": "虚拟机资源大小，Standard_F8s_v2 - 8vcpu、16GiB"
+            }
+        },
+        "networkPolicy": {
+            "type": "string",
+            "defaultValue": "azure",
+            "metadata": {
+                "description": "用于构建 Kubernetes 网络的网络策略。"
+            }
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2020-09-01",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

Reply:
I will recheck soon.